compiler/luci/export/src/CircleOperationExporter.cpp

```diff
@@ -1629,9 +1643,15 @@ void OperationExporter::visit(luci::CircleUnpack *node)
         break;
       }
     }
+    // NOTE real models may not use all of the outputs
     if (!found)
     {
-      INTERNAL_EXN("Invalid Unpack output");
```

Comment:
as a graph, not using all of the outputs is not an error.
using assert is OK but throw is not appropriate.